getnacelle/nacelle-js
# chore: update package-locks after npm i

## WHY are these changes introduced?

Hopefully fixes npm ci runs in GH Actions by including all of the `package-lock` changes generated by newer versions of node/npm.  



## WHAT is this pull request doing?

- Updates lock files after an `lerna clean && lerna bootstrap`.


## How to Test

1. Checkout this branch `mdarrik/shopify-cart/fix-package-lock-issues`
2. Make sure you're using the latest version of node/npm (`npm 8.16.0` & `node v16.16.0`)
3. Clean your node modules by running `npm run clean && rm -rf node_modules` - when prompted in `npm run clean`, type `y`.
4. Run a CI install by running `CI=true npm ci`.  This should trigger Lerna to run in `CI` mode during the postinstall command.
5. Install should finish successfully.
